src/components/ForMore.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+import IconStackoverflow from '../assets/stackoverflow-icon.svg';
+import IconLinkedin from '../assets/linkedin-icon.svg';
+import IconGithub from '../assets/github-icon.svg';
+import { SectionHeading, InnerWrapper } from '../library/GlobalStyles'
+// import { GITHUB_URL, LINKEDIN_URL, STACKOVERFLOW_URL } from '../utils/Urls';
+// <a href={LINKEDIN_URL} target="_blank" rel="noreferrer">
+// <a href={GITHUB_URL} target="_blank" rel="noreferrer">
+// <a href={STACKOVERFLOW_URL} target="_blank" rel="noreferrer">
+
+export const ForMore = () => {
+  return (
+    <InnerWrapper>
+      <SectionHeading>FOR MORE</SectionHeading>
+      <ForMoreIcons>
+        <div className="iconContainer">
+          <img src={IconLinkedin} lang="eng" alt="Linkedin icon" />
+          <p>LINKEDIN</p>
```

Comment:
A nice touch with lang. Such a good idea! I might just steal it from you too hehe